.ci/jobs/elastic+helm-charts+pull-request+cluster-cleanup.yml

```diff
@@ -1,35 +0,0 @@
----
```

Comment:
Will pull-request jobs on master branch be able to test also pull requests for other branches or should we have some pull-request job for each release branch?

Reply:
I'm not sure about the behavior with PRs against this branch with this new config. For now, I have removed those `jjb` files and I would like to first merge this PR and make sure all the `7.7` jobs are OK (CI/staging) before dealing with the PRs jobs